---
title: "Detecting and correcting misclassified sequences in the large-scale public databases"
collection: publications
permalink: /publication/2020-09-01-detecting-misclassified-sequences
excerpt: 'This paper presents a heuristic method to detect and correct potentially misclassified taxonomic assignments in public databases.'
date: 2020-09-01
venue: 'Bioinformatics'
paperurl: 'https://academic.oup.com/bioinformatics/article/36/18/4699/5862012'
citation: 'Hamid Bagheri, Andrew J Severin, Hridesh Rajan, Detecting and correcting misclassified sequences in the large-scale public databases, <i>Bioinformatics</i>, Volume 36, Issue 18, September 2020, Pages 4699â€“4705, https://doi.org/10.1093/bioinformatics/btaa586'
---

**Abstract**

**Motivation**: As the cost of sequencing decreases, the amount of data being deposited into public repositories is increasing rapidly. Public databases rely on the user to provide metadata for each submission, which is prone to user error. Unfortunately, most public databases, such as non-redundant (NR), rely on user input and do not have methods for identifying errors in the provided metadata, leading to the potential for error propagation. Previous research on a small subset of the NR database analyzed misclassification based on sequence similarity. To the best of our knowledge, the amount of misclassification in the entire database has not been quantified. We propose a heuristic method to detect potentially misclassified taxonomic assignments in the NR database. We applied a curation technique and quality control to find the most probable taxonomic assignment. Our method incorporates provenance and frequency of each annotation from manually and computationally created databases and clustering information at 95% similarity.

**Results**: We found more than two million potentially taxonomically misclassified proteins in the NR database. Using simulated data, we show a high precision of 97% and a recall of 87% for detecting taxonomically misclassified proteins. The proposed approach and findings could also be applied to other databases.
